packages/website/src/components/editor/LoadedEditor.tsx

```diff
@@ -222,7 +252,20 @@ export const LoadedEditor: React.FC<LoadedEditorProps> = ({
         }
       }
     };
-  }, []);
+  }, [
+    codeActions,
+    main.languages,
+    onChange,
+    onSelect,
+    sandboxInstance.editor,
+    sandboxInstance.monaco.editor,
+    sandboxInstance.monaco.languages.json.jsonDefaults,
+    tabs.code,
+    tabs.eslintrc,
+    tabs.tsconfig,
+    updateMarkers,
+    webLinter.ruleNames,
+  ]);
```

Comment:
There are still two deps complaints later on in this file: line 283 and line 292. Did you mean to tackle those also?

Reply:
The missing dependencies were updated on commit 59b66ac748793f9537dff14855cf5b94c2fb2591.